### ISSUE TITLE
$input-btn-border-width on bootstrap 5.3 is a custom variable (on 5.2 was a simple scss variable)

### ISSUE THREAD

**Issue by meriturva** [telerik/kendo-themes#4535]
See here:

https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#LL804C1-L804C24

`$input-btn-border-width` is now a custom variable.

(see bootstrap 5.2 version: https://github.com/twbs/bootstrap/blob/v5.2.3/scss/_variables.scss#L738 where `$input-btn-border-width` was a simple scss variable)

The main problem is that on file:

[`@progress\kendo-theme-bootstrap\scss\input\_variables.scss `](https://github.com/telerik/kendo-themes/blob/c16c3f6d10bf78fe8b8a5ea1799e473557b04a01/packages/bootstrap/scss/input/_variables.scss#LL9C1-L9C1) you are using:

```
$kendo-input-border-width: $input-border-width !default;
$kendo-input-border-height: ( $kendo-input-border-width * 2 ) !default;
```
so scss compilation gives me an error:

```
SassError: Undefined operation "var(--bs-border-width) * 2".
   ╷
10 │ $kendo-input-border-height: ( $kendo-input-border-width * 2 ) !default;
```
My temporary fix is to define, before importing _variables file:

`$kendo-input-border-width:      1px !default;`

Basically, the main problem is that package @progress/kendo-theme-bootstrap 6.4.0 with bootstrap 5.3 doesn't work anymore.

Feedback item: https://feedback.telerik.com/themes/1611266

Diego

**Comment by Camo30** [comment]
Same here. Seems that this package is not yet compatible with bootstrap 5.3, tested up to recent version 5.3.2.
Temporary fix is to downgrade to bootstrap 5.2.3.

**Comment by vveesseelliinnaa** [comment]
Reported again in Ticket ID: 1627595

**Comment by AStoker** [comment]
Running into a similar issue that appears related. Same kind of error, but regarding the `$kendo-button-border-width` in the multiselect element.

```
Undefined operation "var(--bs-border-width) * 2".
   ╷
69 │ $kendo-button-calc-size: calc( #{$kendo-button-line-height * 1em} + #{$kendo-button-padding-y * 2} + #{$kendo-button-border-width * 2} ) !default;
   │                                                                                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
  node_modules\@progress\kendo-theme-bootstrap\scss\button\_variables.scss 69:104  @import
  node_modules\@progress\kendo-theme-bootstrap\scss\button\_index.scss 21:9        @import
  @progress\kendo-theme-bootstrap\scss\multiselect\_index.scss 21:9                @import
```

Downgrading bootstrap to specifically 5.2.3 (was 5.3.3) fixes me for now, but that's not a real solution (especially for a ticket that's been open for months now).

**Comment by vveesseelliinnaa** [comment]
Reported again in Ticket ID: 1666510

**Comment by oscarmanzelli** [comment]
My ticket ID is [1688960](https://www.telerik.com/account/support-center/view-ticket)

**Comment by TeyaVes** [comment]
no longer reproducible as of Kendo Bootstrap v. 12.0.0- the $kendo-button-border-width and $kendo-input-border-width variables are now with a set value:

https://github.com/telerik/kendo-themes/blob/5c5fce081dad55f2f851db67db1e7f2c401e5987/packages/bootstrap/scss/button/_variables.scss#L11

https://github.com/telerik/kendo-themes/blob/5c5fce081dad55f2f851db67db1e7f2c401e5987/packages/bootstrap/scss/input/_variables.scss#L15
